static/js/src/advantage/distributor/components/ChannelOffersList/ChannelOffersList.tsx

```diff
@@ -74,10 +75,64 @@ const ChannelOffersList = () => {
     },
   );
 
+  const searchOfferList = sortedOffersList?.filter((offer: OfferType) => {
+    const opId = offer?.external_ids?.[0]?.ids?.[0];
+    const opportunityNumber = offer?.opportunity_number;
+    const creatorName = offer?.channel_deal_creator_name;
+    const resellerName = offer?.reseller_account_name;
+    const customerName = offer?.end_user_account_name;
+
+    return (
+      opId?.toLowerCase().includes(searchValue.toLowerCase()) ||
```

Comment:
Can we extract `searchValue.toLowerCase()` to a constant so it's less repetitive?